openedx/core/djangoapps/xblock/utils.py

```diff
@@ -46,11 +46,69 @@ def get_secure_token_for_xblock_handler(user_id, block_key_str, time_idx=0):
     token each user gets for a given XBlock doesn't change, which makes
     debugging and reasoning about the system simpler.)
     """
+    # Impact of exposer of the SECRET_KEY or XBLOCK_HANDLER_TOKEN_KEYS:
+    # An actor with the key could produce valid secure tokens for any given user_id and xblock and
+    # could submit answers to questions on behalf of other users. This would violate the integrity
+    # of the answer data on our platform.
+    # Estimated CVSS Base Score: 7.5
+    # Score URL: https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N
+    # Environment score not given because the importance of the environment may change over time.
+
+    # Fall back to SECRET_KEY if XBLOCK_HANDLER_PEPPERS is not set or is an empty list.
+    # If this key is actually exposed to malicious actors, we should do a rapid rotation that
+    # breaks people because in this case a malicious actor can generate valid tokens to submit
+    # answers as any user.
+    # Transitioning from SECRET_KEY to XBLOCK_HANDLER_TOKEN_KEYS:
+    #
+    # 1. Add the current secret key and a new xblock handler specific secret key to the
+    # XBLOCK_HANDLER_TOKEN_KEYS list in your settings file or yml. The order of the keys
+    # matters and so the new xblock specific key should be at index 0.
+    #   eg. XBLOCK_HANDLER_TOKEN_KEYS = ["<new xblock specific hashing key>", "<value of django secret key>"]
+    # 2. Wait 4 days after the code has been deployed to production.
+    # 3. Remove the django secret key from the XBLOCK_HANDLER_TOKEN_KEYS list.
+    #   eg. XBLOCK_HANDLER_TOKEN_KEYS = ["<new xblock specific hashing key>"]
+
+    # Rotation Process(Assumes you've already transitioned from SECRET_KEY to XBLOCK_HANDLER_TOKEN_KEYS):
```

Comment:
```suggestion
    # Rotation Process (Assumes you've already transitioned from SECRET_KEY to XBLOCK_HANDLER_TOKEN_KEYS):
```
nit: missing whitespace